src/daemon/daemon.pro

```diff
@@ -5,7 +5,7 @@ QT -= gui
 QT += dbus
 
 CONFIG += \
-        c++11 \
+        c++17 \
```

Comment:
At least when i make apps for SFOS, their Qt version does not recognize c++17 as CONFIG, so i had to resort to `QMAKE_CXXFLAGS += -std=c++17`.
(And c++11 maps to `-std=gnu++0x`)